contrib/builder/rpm/armhf/centos-7/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM multiarch/centos:7.2.1511-armhfp-clean
+
+RUN yum groupinstall --skip-broken -y "Development Tools"
+RUN yum -y swap -- remove systemd-container systemd-container-libs -- install systemd systemd-libs
+RUN yum install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel pkgconfig selinux-policy selinux-policy-devel sqlite-devel systemd-devel tar git
+
+ENV GO_VERSION 1.7.3
+RUN curl -fSL "https://storage.googleapis.com/golang/go${GO_VERSION}.linux-armv6l.tar.gz" | tar xzC /usr/local
```

Comment:
can you change this url to https://golang.org/dl/go${GO_VERSION}.linux-armv6l.tar.gz to be consistent with all the other ones.